### ISSUE TITLE
iOS 14 : dev menu shows "Information not available"

### ISSUE THREAD

**Issue by sebsto** [aws-amplify/amplify-ios#850]
**Describe the bug**
I added a DevMenu as per Amplify iOS 1.4.0

On iOS Simulator :
When triggering the DevMenu, the 'Amplify Plugins Information' an 'Developer Environment Information' says 
"version not available" or "information not available"
Also, Log viewers reports "Log Unavailable"

**To Reproduce**
Implement Dev Menu as this 
https://github.com/sebsto/amplify-ios-getting-started/commit/70a42197ebc2975cfb7d6761d9261d759e7f570c

Trigger Dev Menu with a long touch gesture 

Navigate to "environment information"
Navigate to "log viewer"

**Expected behavior**
DevMenu should report valuable data for debugging

**Screenshots**
![Simulator Screen Shot - iPhone 11 - 2020-10-23 at 07 41 03](https://user-images.githubusercontent.com/401798/96960624-208b0900-1503-11eb-817f-0daafcd15b27.png)
![Simulator Screen Shot - iPhone 11 - 2020-10-23 at 07 41 21](https://user-images.githubusercontent.com/401798/96960638-284aad80-1503-11eb-92d6-8c802de67f64.png)

**Environment(please complete the following information):**
 - Amplify Framework Version: 1.4.0
 - Dependency Manager: cocoapods
 - Swift Version : Xcode 12.1 / Swift 5.?

**Device Information (please complete the following information):**
 - Device: simulator
 - iOS Version: 14.1

**Additional context**

**Comment by sebsto** [comment]
Just tested on device - same behaviour

**Comment by drochetti** [comment]
hey @sebsto, thanks for testing this as soon as it came out 🙂 

We [published the docs this morning](https://docs.amplify.aws/lib/debugging/dev-menu/q/platform/ios). I'm working on an update to reflect the new SwiftUI App life cycle, since this was created before that came out.

---

The lack of version is something that I'm fixing right now. As for the logs, you should be able to see it. Can you clarify if you enabled the menu before calling `Amplify.configure()`?

**Comment by sebsto** [comment]
Thank you for the follow up. I saw the doc PR being merged yesterday.
Here are the steps I took (aligned with what I read in the doc) 

1. Add `local-env-info.json` to the project 
2. Add ènableDevMenu`as this : 
```swift 
           try Amplify.add(plugin: AWSCognitoAuthPlugin())
           try Amplify.add(plugin: AWSAPIPlugin(modelRegistration: AmplifyModels()))
           try Amplify.add(plugin: AWSS3StoragePlugin())
            
           Amplify.enableDevMenu(contextProvider: self)
           try Amplify.configure()
```

3. and implement `DevMenuPresentationContextProvider`  as this 

```swift 
    func devMenuPresentationContext() -> UIWindow {
        UIApplication.shared.windows.first(where: \.isKeyWindow)!
    }
```

**Comment by sebsto** [comment]
The file is correctly bundled in the package :

```zsh
➜  code git:(main) ✗ cd /Users/stormacq/Library/Developer/CoreSimulator/Devices/D00C65B6-6240-4903-A84F-945E6177BED3/data/Containers/Bundle/Application/DF03C80B-0E60-4658-89A3-E52B8F19C48C/getting\ started.app 
➜  getting started.app ls
Base.lproj                Info.plist                _CodeSignature            awsconfiguration.json     local-env-info.json
Frameworks                PkgInfo                   amplifyconfiguration.json getting started           schema.graphql
➜  getting started.app cat local-env-info.json 
{
    "projectPath": "/Users/stormacq/Documents/amazon/code/amplify/ios-getting-started/code",
    "envName": "amplify"
} 
```

**Comment by palpatim** [comment]
Still investigating

**Comment by palpatim** [comment]
The plugin information will be fixed in the next release. The environment info will be populated as we enhance the CLI to provide the various data points.

**Comment by wooj2** [comment]
This was released as part of 1.5.0.  Optimistically closing this issue, but if you have any questions/problems regarding this issue, feel free to re-open.

**Comment by sebsto** [comment]
Tested with Amplify 1.5.0 runtime libs, and an "old" project created with the CLI from 2/3 months ago. Still not working. Is there a requirement to create the project with a recent CLI ?

**Comment by palpatim** [comment]
@sebsto D'oh!

I did a late-breaking change to add a default implementation of to `AmplifyVersionable` for types conforming to `AnyObject`, but *also* removed the Conformance of each plugin to `AmplifyVersionable`. I thought I had tested the menu after this change, but apparently I didn't do so properly. Will fix in an upcoming PR.

**Comment by royjit** [comment]
The fix is released in 1.5.2, please take a look https://github.com/aws-amplify/amplify-ios/releases/tag/1.5.2

**Comment by sebsto** [comment]
I confirm 1.5.2 fixes this problem - thanks
